### ISSUE TITLE
Flow type arguments at callsites in class bodies

### ISSUE THREAD

**Issue by rattrayalex-stripe** [michaelgmcd/vscode-language-babel#43]
[This flow language feature](https://flow.org/en/docs/types/generics/#toc-supplying-type-arguments-to-callables) is broken in class bodies (and possibly elsewhere, I haven't looked):

![screen shot 2019-02-28 at 2 40 39 pm](https://user-images.githubusercontent.com/29717832/53605079-4368ec80-3b6b-11e9-913a-de10692aff5a.png)
```jsx
class Correct extends React.Component<MyProps> {
  myRef = React.createRef();

  render() {
    const {foo} = this.props;
    return null;
  }
}

class NotCorrect extends React.Component<MyProps> {
  myRef = React.createRef<HTMLDivElement>(); // This causes the problem!

  render() {
    const {foo} = this.props;
    return null;
  }
}
```

**Comment by mgtitimoli** [comment]
I can confirm it's also broken at function calls on every context (class body and everywhere else).

**Comment by gastonmorixe** [comment]
+1

**Comment by michaelgmcd** [comment]
The `>` character is the issue. When inspecting the TM scope, it seems like it's being treated as the `>` of an arrow function (meta.function.arrow.js) which seems to break all of the syntax highlighting until the next `}` (end of the make believe function. This might be a little tricky to debug, but I'll try to take a look when I can.

**Comment by michaelgmcd** [comment]
Please check that this was fixed in v0.0.24

**Comment by rattrayalex-stripe** [comment]
Yep, looks perfect!
<img width="832" alt="Screen Shot 2019-09-11 at 8 34 59 AM" src="https://user-images.githubusercontent.com/29717832/64711817-1331cd00-d46f-11e9-87d6-56b5fb42ad26.png">
